console/HAL-1981_Dashboard.adoc

```diff
@@ -0,0 +1,172 @@
+---
+categories:
+- console
+---
+= [Experimental] Console Dashboard
+:author:            Harald Pehl
+:email:             hpehl@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+A dashboard will replace the homepage of the current console. The dashboard should visualize WildFly's most essential data and follow the guidelines from the PatternFly website: https://www.patternfly.org/patterns/dashboard/design-guidelines.
+
+.Dashboard sample
+image::https://www.patternfly.org/images/dashboard-view-desktop.d9b02461450d9a4e.png[Dashboard sample]
+
+Different cards visualize the data on the dashboard. These cards include data like:
+
+* Name, version, build, and operation mode of WildFly.
+* Runtime information about memory, threads, and other essential resources.
+* MicroProfile health status
+* Information about deployments
+* The last <n> log statements (with errors and warnings highlighted)
+* Links to documentation, help, forums, and information on how to get involved with WildFly.
+
+== Issue Metadata
+
+The feature will be part of the next-gen console (https://github.com/hal/foundation/). It won't be available in the current (3.x) codebase.
+
+=== Issue
+
+* https://issues.redhat.com/browse/HAL-1981[HAL-1981]
+
+=== Stability Level
+
+* [x] Experimental
+
+* [ ] Preview
+
+* [ ] Community
+
+* [ ] default
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== Testing By
+
+* [x] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+* Web Console
+
+=== Relevant Installation Types
+
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [x] Managed domain
+
+* [ ] OpenShift s2i
+
+* [ ] Bootable jar
+
+== Requirements
+
+The dashboard visualizes WildFly's most essential data, including configuration and runtime data and links to valuable resources. Different cards visualize the data on the dashboard. If a card shows dynamic data, it includes a refresh button. In addition, all cards are refreshed automatically (for static cards this is a noop).
+
+=== Cards
+
+==== Product info
+
+This card shows WildFly's static and configuration data. This includes name, version, operation mode, stability level, OS/CPU, and JVM information.
+
+*Sources*
+
+* `:read-resource(attributes-only,include-runtime)`
```

Comment:
thinking out loud but should this card be dynamic to track changes in the running mode of the servers?

Reply:
I'd like to keep this static and move the running mode of the servers in the domain resp. runtime card.